# Possible game states

uint8 FREE_KICK = 0
uint8 PENALTY_KICK = 1
uint8 GOAL_KICK = 2
uint8 FREE_BALL = 3
uint8 KICKOFF = 4
uint8 STOP = 5
uint8 GAME_ON = 6
uint8 HALT = 7

uint8 data
